tests/TestCase/Command/HelpCommandTest.php

```diff
@@ -37,7 +37,7 @@ public function setUp()
     {
         parent::setUp();
         $this->setAppNamespace();
-        $this->useCommandRunner(true);
+        $this->useCommandRunner();
```

Comment:
I think we can use commandRunner by default with changing
 https://github.com/cakephp/cakephp/blob/8baeda92fbccc426608a2d347ead7f5dfccb5af1/src/TestSuite/ConsoleIntegrationTestTrait.php#L41

Reply:
I'm saying we can remove whole `$this->useCommandRunner();` if we would change the
`$_useCommandRunner = false;` to `$_useCommandRunner = true;`
since shell will remove in v5

Reply:
Feel free to make a PR in top here.

Reply:
I'll open , after merging this PR